boto/iam/connection.py

```diff
@@ -1743,6 +1743,56 @@ def create_virtual_mfa_device(self, path, device_name):
         }
         return self.get_response('CreateVirtualMFADevice', params)
 
+    def delete_virtual_mfa_device(self, serial_number):
+        """
+        Deletes a virtual MFA device for the AWS account.
+        You must deactivate a user's virtual MFA device
+        before you can delete it.
+
+        :type serial_number: string
+        :param serial_number: The serial number which uniquely identifies
+            the MFA device. For virtual MFA devices, the serial number is
+            the same as the ARN.
+        """
+        params = {
+            'SerialNumber': serial_number
+        }
+        return self.get_response('DeleteVirtualMFADevice', params)
+
+    def get_all_virtual_mfa_devices(self, assignment_status=None, marker=None, max_items=None):
+        """
+        Lists the virtual MFA devices defined in the AWS account
+        by assignment status. If you do not specify an assignment status,
+        the operation returns a list of all virtual MFA devices.
+        Assignment status can be 'Assigned', 'Unassigned', or 'Any'.
+
+        :type assignment_status: string
+        :param assignment_status: The status (Unassigned or Assigned) of
+             the devices to list. If you do not specify an AssignmentStatus,
+             the operation defaults to Any, which lists both assigned and
```

Comment:
Let's changed the doc here...

> If you do not specify an AssignmentStatus, the operation lists both assigned and unassigned virtual MFA devices. You can also explicitly 'Any' which returns the same thing.

Reply:
Thanks. Updated here. https://github.com/cloudian/boto/pull/23/commits/b6364e193ecd5096cb190e5d7ce64adb89cf536b